eth2/operation_pool/src/attestation.rs

```diff
@@ -40,15 +58,18 @@ impl<'a, T: EthSpec> MaxCover for AttMaxCover<'a, T> {
     fn update_covering_set(
         &mut self,
         best_att: &Attestation<T>,
-        covered_validators: &BitList<T::MaxValidatorsPerCommittee>,
+        covered_validators: &HashMap<u64, u64>,
     ) {
         if self.att.data.slot == best_att.data.slot && self.att.data.index == best_att.data.index {
-            self.fresh_validators.difference_inplace(covered_validators);
+            for key in covered_validators.keys() {
+                let _ = self.fresh_validators_rewards.remove(key);
+            }
         }
     }
 
     fn score(&self) -> usize {
-        self.fresh_validators.num_set_bits()
+        let size: u64 = self.fresh_validators_rewards.values().sum();
+        (size as u64).try_into().unwrap()
```

Comment:
Anyway of getting through this without using try_into? I believe we'd have some problems on 32bits machines here if size gets really large... ?

Reply:
We normally just cast between `u64` and `usize` assuming they're the same size. So `size as usize` is fine.

This is one of the reasons we're "64 bit only" at the moment.

Reply:
Or you could make it a one liner `self.fresh_validators_rewards.values().sum::<u64>() as usize`